docs/extensions/outlook-integration/troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+# Outlook troubleshooting
+
+In this article:
+
+* [Check scheduled jobs](#check-scheduled-jobs)
+* [Check logs](#check-logs)
+* [Server side error 500: Could not get access token for Outlook](#server-side-error-500-could-not-get-access-token-for-outlook)
+* [Application is not configured as a multi-tenant application](#error-application-is-not-configured-as-a-multi-tenant-application)
+* [Regular user's issues](#regular-users-issues)
+* [Max number of operations](#max-number-of-operations)
+
+## Check Scheduled Jobs
+
+Check whether the scheduled job is running in Administration -> Scheduled Jobs -> Outlook Calendar Sync -> Log
```

Comment:
* `>` instead of `->`
* period `.` in the end of the sentence.
* `at` instead of `in`.